examples/meta/src/neural_nets/feedforward_net_regression.sg

```diff
@@ -1,54 +1,44 @@
-CSVFile f_feats_train("../../data/regression_1d_sinc_features_train_with_9d_noise.dat")
-CSVFile f_feats_test("../../data/regression_1d_sinc_features_test_with_9d_noise.dat")
-CSVFile f_labels_train("../../data/regression_1d_sinc_labels_train.dat")
-CSVFile f_labels_test("../../data/regression_1d_sinc_labels_test.dat")
+File f_feats_train = csv_file("../../data/regression_1d_sinc_features_train_with_9d_noise.dat")
+File f_feats_test = csv_file("../../data/regression_1d_sinc_features_test_with_9d_noise.dat")
+File f_labels_train = csv_file("../../data/regression_1d_sinc_labels_train.dat")
+File f_labels_test = csv_file("../../data/regression_1d_sinc_labels_test.dat")
 
 Math:init_random(1)
 
 #![create_features]
 Features features_train = features(f_feats_train)
 Features features_test = features(f_feats_test)
-RegressionLabels labels_train(f_labels_train)
-RegressionLabels labels_test(f_labels_test)
+Labels labels_train = labels(f_labels_train)
+Labels labels_test = labels(f_labels_test)
 #![create_features]
 
-#![add_layers]
-int dimensions = features_train.get_int("num_features")
-NeuralLayers layers() 
-layers.input(dimensions)
-layers.rectified_linear(20)
-layers.linear(1)
-DynamicObjectArray all_layers = layers.done()
```

Comment:
Is this taken care of?

Reply:
ah yes it just returns the DynamicObjectArray so if we set that instead using add, all should be good